R/new_matrixfit.R

```diff
@@ -402,22 +422,29 @@ new_matrixfit <- function(cf,
                bsamples = cf$cf.tsboot$t[, ii],
                use.minpack.lm = fit.method == 'lm',
                error = cf$error_fn,
-               cov_fn = cf$cov_fn)
-  
+               cov_fn = cf$cov_fn,
+               priors = priors,
+               ...)
+
   if (useCov) {
-    args$CovMatrix <- cf$cov_fn(cf$cf.tsboot$t[, ii])
+    if(!missing(priors)){
+      args$CovMatrix <- cf$cov_fn(cbind(cf$cf.tsboot$t[, ii], priors$psamples))
```

Comment:
Does that make sense? This would compute the correlation between the parameters that came out of the fit and the priors that we put in. Why is that useful?

Reply:
As discussed with @martin-ueding this is correct.